evalml/automl/automl_search.py

```diff
@@ -764,6 +757,18 @@ def _post_evaluation_callback(self, pipeline, evaluation_results):
             self.add_result_callback(self._results['pipeline_results'][pipeline_id], pipeline, self)
         return pipeline_id
 
+    def _check_for_high_variance(self, pipeline, cv_scores, threshold=0.2):
+        """Checks cross-validation scores and logs a warning if variance is higher than specified threshhold."""
+        pipeline_name = pipeline.name
+        high_variance_cv = False
+
+        if cv_scores.mean() != 0:
+            high_variance_cv = bool(abs(cv_scores.std() / cv_scores.mean()) > threshold)
```

Comment:
Isn't `abs(cv_scores.std() / cv_scores.mean()) > threshold` already a bool?

Reply:
@freddyaboulton Unfortunately it resolves to a `numpy.bool_` which is different 😭

Reply:
Damn. But then how does the old code work? 🤔

Reply:
Old code (and this code) check `if high_variance_cv` which will evaluate to True regardless of python or numpy bool, but now we want to take `high_variance_cv` and set it in `pipeline_results`. (Previously, just checked if # of warnings greater than 0, then set `high_variance_cv` to True so not using it directly.)

I wasn't aware of the difference myself until `test_search_results` broke at `assert isinstance(results['high_variance_cv'], bool)` without this type conversion 😬